src/sentry/auth/idpmigration.py

```diff
@@ -1,6 +1,35 @@
-import uuid
+import string
+from datetime import timedelta
 
-from sentry.models import Organization, User
+from django.utils.crypto import get_random_string
+
+from sentry import options
+from sentry.models import Organization, OrganizationMember, User
+from sentry.utils import redis
+from sentry.utils.email import MessageBuilder
+
+_REDIS_KEY = "verificationKeyStorage"
+_TTL = timedelta(minutes=10)
+
+
+def send_confirm_email(user: User, email: str, verification_key: str) -> None:
+    context = {
+        "user": user,
+        # TODO left incase we want to have a clickable verification link for future
+        # "url": absolute_uri(
+        #     reverse("sentry-account-confirm-email", args=[user.id, verification_key])
+        # ),
+        "confirm_email": email,
+        "verification_key": verification_key,
+    }
+    msg = MessageBuilder(
+        subject="{}Confirm Email".format(options.get("mail.subject-prefix")),
+        template="sentry/emails/idp_verification_email.txt",
+        html_template="sentry/emails/idp_verification_email.html",
+        type="user.confirm_email",
+        context=context,
+    )
+    msg.send_async([email])
 
 
 def create_verification_key(user: User, org: Organization, email: str) -> None:
```

Comment:
i'm wondering if we could rename this method to be more descriptive about sending an email to the user: `create_verification_key` -> `send_one_time_account_confirm_link`?